tools/ci_set_matrix.py

```diff
@@ -150,7 +152,7 @@ def get_settings(board):
                 continue
 
             # Boards don't run tests or docs so ignore those as well.
```

Comment:
It's nitpicky but do you mind updating the comment and/or moving it to where IGNORE_DIRS is defined?

It might say
```py
# Files in these directories never influence board builds
IGNORE_DIRS = [ ... ]
```
and then no comment down at the `if any ...: continue` line would be needed imho.

Reply:
Valid point, I will update that.